packages/react-components/source/scss/library/components/forms/_input.scss

```diff
@@ -46,6 +46,22 @@
   box-shadow: 0 2px 0 0 $puppet-b300 !important;
 }
 
+.rc-input-squared {
+  border-radius: $puppet-common-border-radius;
+}
+
+.rc-input-rounded {
+  border-radius: 81px;
```

Comment:
this might be intentional on the designer's part. but if i can't figure out a good reason why a stylesheet number isn't a multiple of 4 I round it.